report_qweb_signer/models/report.py

```diff
@@ -145,7 +147,8 @@ def pdf_sign(self, pdf, certificate):
             raise Warning(
                 _('Signing report (PDF)'),
                 _('jPdfSign failed (error code: %s). '
-                  'Message: %s') % (str(process.returncode), err))
+                  'Message: %s. Output: %s') %
+                (str(process.returncode), err, out))
```

Comment:
[`str()` is implicit with `%s`](https://docs.python.org/2/library/stdtypes.html#string-formatting-operations) AFAIK.